.eslintignore

```diff
@@ -6,6 +6,9 @@ packages/react-art/npm/lib
 
 # Build products
 build/
+# TODO: Currently storing artifacts as `./build2` so that it doesn't conflict
+# with old build job. Remove once we migrate rest of build/test pipeline.
```

Comment:
Comment is copied from https://github.com/facebook/react/blob/c06d245fc7325612347ed19232d92385cae2e0a7/scripts/rollup/build-all-release-channels.js#L45-L47